htdocs/stripe/class/stripe.class.php

```diff
@@ -549,11 +543,10 @@ public function getPaymentIntent($amount, $currency_code, $tag, $description = '
 	 * @param	string 	$customer							Stripe customer ref 'cus_xxxxxxxxxxxxx' via customerStripe()
 	 * @param	string	$key							    ''=Use common API. If not '', it is the Stripe connect account 'acc_....' to use Stripe connect
 	 * @param	int		$status							    Status (0=test, 1=live)
-	 * @param	int		$usethirdpartyemailforreceiptemail	1=use thirdparty email for receipt
```

Comment:
Why removing this parameter ?

Reply:
bad remove, between payment intetn et setupintent ... i will fix it

Reply:
@eldy  setupintent is no payment intent, it's just for register payment method for later use (signe ou multiuse), receipt_email don't exist

Reply:
cf https://stripe.com/docs/api/setup_intents/object#setup_intent_object-id